src/index.js

```diff
@@ -5,5 +5,8 @@ export const createToken = name => () => {
 export const createOptionalToken = (name, defaultValue) => () => defaultValue;
 
 // Tokens
-export const GenericSessionToken = createToken('GenericSession');
-export const FetchToken = createToken('Fetch');
+export const FetchToken = createToken('FetchToken');
+export const LoggerToken = createToken('LoggerToken');
+export const ReduxToken = createToken('ReduxToken');
+export const SessionToken = createToken('SessionToken');
+export const EventEmitterToken = createToken('EventEmitterToken');
```

Comment:
I think we should call this `UniversalEvents` token. Also, all of these need type definitions